### DISCUSSION TITLE
Do the credits look good?

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#671] [General]
A while back I modified the colors used in the credits to make them easier to read.  I also added some folks mentioned in a text file on the CD that hadn't made it into credits in the game, and I added a page for us.  Some of this is "best guess" on my part, so I'd like to know if there's anything here I should add or change.  The credits for who put out the various releases will change, of course, depending who does them for 2.0.  (Unless people think that shouldn't be in the credits in the first place.)

![Credits1](https://user-images.githubusercontent.com/3118014/187499252-7d02452a-f9a5-4e42-bf14-6c87a574acfd.png)

**Comment by zombiedesigner** [comment]
Really nice!
